shadow-cljs.edn

```diff
@@ -1,9 +1,16 @@
 {:dev-http {3000 "public"}
  :source-paths ["src" "test" "dev"]
- :builds {:app {:target :browser
+ :dependencies [[applied-science/js-interop "0.4.2"]]
+ :builds {:app {:target :esm ; can be loaded in < script > with `type="module"`
```

Comment:
changing from `:target :browser` to `:target :esm` isn't strictly necessary, but I figured it would be good for consistency. That and, I came across [this message](https://clojurians.slack.com/archives/C6N245JGG/p1761673829814219?thread_ts=1761597522.189239&cid=C6N245JGG) from Thomas Heller on Slack:

> I should really write an updated code splitting guide for esm as well. generally :esm is much better than :browser pretty much always